usr/lib/common/loadsave.c

```diff
@@ -2848,6 +2848,14 @@ CK_RV load_public_token_objects(STDLL_TokData_t *tokdata)
             continue;
         }
 
+        /* size can not be negative if treated as signed int */
```

Comment:
as unsigned int

Reply:
no, the problem arises if it is treated as **signed** int, because then its value would be negative.